gst/datarepo/gstdatareposrc.c

```diff
@@ -198,12 +199,24 @@ gst_data_repo_src_class_init (GstDataRepoSrcClass * klass)
           GST_PARAM_MUTABLE_READY));
 
   g_object_class_install_property (gobject_class, PROP_EPOCHS,
-      g_param_spec_int ("epochs", "Epochs",
+      g_param_spec_uint ("epochs", "Epochs",
           "Repetition of range of files or samples to read, set number of repetitions",
           0, G_MAXINT, DEFAULT_EPOCHS,
```

Comment:
`G_MAXUINT` ?

Reply:
Thank you.
property can get uint value but.
Limit the number of samples to a number in the range int.
However, in the case of epochs, it doesn't seem to make much sense.